Docs/Configuration.tex

```diff
@@ -2549,9 +2548,9 @@ \subsection{Scheme Properties}\label{kernelpropsscheme}
   \texttt{Mkext}, \texttt{Prelinked}) when available.
 
   Different variants of macOS support different kernel caching variants designed to improve
-  boot performance. This setting allows to prevent using faster kernel caching variants
-  if slower variants are available for debugging and stability reasons. I.e. by
-  specifying \texttt{Mkext} one will disable \texttt{Prelinked} for e.g. 10.6 but not 10.7.
+  boot performance. This setting prevents the use of faster kernel caching variants
+  if slower variants are available for debugging and stability reasons. I.E., by
```

Comment:
I think it should be `I.e.`, not `I.E.`. Some [websites](https://ell.stackexchange.com/questions/56861/capitalize-i-e-and-e-g) agree at least.

Reply:
Noted